sycl/test/syclcompat/launch/kernel_properties.cpp

```diff
@@ -23,7 +23,7 @@
 // We need hardware which can support at least 2 sub-group sizes, since that
 // hardware (presumably) supports the `intel_reqd_sub_group_size` attribute.
 // REQUIRES: sg-32 && sg-16
-// RUN: %{build} %if cl_options %{/clang:-S /clang:-emit-llvm%} %else %{-S -emit-llvm%} -o - | FileCheck %s
```

Comment:
@ayylol I'd really appreciate your advice here. I rebased after https://github.com/intel/llvm/pull/15744 was merged. I have removed the `%if cl_options` clause here, because I *think* `%clangxx` (and the fact I've moved the test from `sycl/test-e2e` to `sycl/test`) means that I don't need it, but I'm not confident about that. Is this OK or am I undoing your work?

Reply:
Hey @joeatodd I am not so familiar with the how the tests in `test` function to know right away if this would not cause any issues, however I can describe what was needed to reproduce the issue when it was in the e2e folder so you can check if it is needed after you move it.

in `test-e2e` the compiler used in `%clangxx` is set via the [CMAKE_CXX_COMPILER](https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/CMakeLists.txt#L19-L25) flag. Setting this to `clang-cl` on windows is what made the warning be emitted, due to the `-emit-llvm` flag not existing on the MSVC driver. Because the e2e tests also add the `-Werror` flag to the `%clangxx` expansion this created a failure.

Reply:
Thanks for the info :pray: